blocks/form/form.css

```diff
@@ -476,6 +476,16 @@ main .form form div[data-variant="addDeleteButtons"] .item-remove {
     display: block;
 }
 
+/* Hide add button when at max limit */
+main .form form .repeat-wrapper[data-at-max="true"] .item-add {
```

Comment:
Please try to rename this to something better `data-add-instance="false"` `data-remove-instance="false"`